app/javascript/components/ingredients/ingredients-container.vue

```diff
@@ -227,8 +230,12 @@ export default {
     async editIngredient() {
       this.showingEdit = !this.showingEdit;
       try {
-        const res = await editIngredient(this.ingredientToEdit.id, this.$refs.editIngredientInfo.form);
-        this.updateIngredient(res);
+        var ingredientsInfo = this.$refs.editIngredientInfo.form;
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  All 'var' declarations must be at the top of the function scope              vars-on-top